source/features/tag-changelog-link.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+Adds a compare link on each releases/tags/single tag page so that you can see what has changed since the previous release.
+If the tags are namespaced then it tries to get the previous release of the same namespaced tag.
+
+See it in action at: https://github.com/parcel-bundler/parcel/releases
+*/
+import './tag-changelog-link.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {diff} from '../libs/icons';
```

Comment:
```suggestion
import * as diff from '../libs/icons';
```

I think it's clearer to use this as `icons.diff()` later later than `diff()`